### DISCUSSION TITLE
群晖上用docker安装aliyundrive-webdav启动失败

### DISCUSSION THREAD

**Discussion by Wccczy** [messense/aliyundrive-webdav#576] [Q&A] [answered]
版本： docker中选择:latest
在可视化界面配置好参数之后运行又一下错误，请懂的大神能指点一下是不是哪里配置错了

 Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: 

Error creating bean with name 'aliYunDriverCronTask': Unsatisfied dependency expressed through field 'aliYunDriverClientService'; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'aliYunDriverClientService' defined in URL [jar:file:/webdav.jar!/BOOT-INF/classes!/com/github/zxbu/webdavteambition/store/AliYunDriverClientService.class]: 

Unsatisfied dependency expressed through constructor parameter 0; 
nested exception is org.springframework.beans.factory.BeanCreationException: 
Error creating bean with name 'teambitionClient' defined in class path resource 

[com/github/zxbu/webdavteambition/config/AliYunDriverAutoConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate 

[com.github.zxbu.webdavteambition.client.AliYunDriverClient]: Factory method 'teambitionClient' threw exception; nested exception is java.lang.NullPointerException

**Comment by messense** [comment, marked as answer]
非本项目问题，你这个是 Java 版的，你看到这个项目用 Java 了吗？
